fiboa_cli/datasets/nl.py

```diff
@@ -0,0 +1,101 @@
+from ..convert_utils import convert as convert_
+
+URI = "https://service.pdok.nl/rvo/referentiepercelen/atom/downloads/referentiepercelen.gpkg"
+
+ID = "nl"
+TITLE = "Field boundaries for The Netherlands"
+DESCRIPTION = """
+A field block (Dutch: "Referentieperceel"), formerly known as "AAN" (Agrarisch Areaal Nederland),
+is a contiguous agricultural area surrounded by permanent boundaries, which is cultivated by one or
+more farmers with one or more crops, is fully or partially set aside or is fully or partially
+taken out of production.
+
+The following field block types exist:
+
+- Woods (Hout)
+- Agricultural area (Landbouwgrond)
+- Other (Overig)
+- Water (Water)
+
+We filter on "Agricultural area" in this converter.
+For crop data, look at BasisRegistratie gewasPercelen (BRP)
+"""
+BBOX = [2.35417303, 50.71447164, 7.5553525, 55.66948102]
+
+PROVIDER_NAME = "RVO / PDOK"
+PROVIDER_URL = "https://www.pdok.nl/introductie/-/article/referentiepercelen"
+ATTRIBUTION = None
+LICENSE = "CC0"  # http://creativecommons.org/publicdomain/mark/1.0/
```

Comment:
http://creativecommons.org/publicdomain/mark/1.0/ is different from CC0, it seems.
I can't find the PDM 1.0 on https://spdx.dev/use/specifications/ so this may need a link as explained in the template.
What's the correct license?

Reply:
Oh, I wasn't aware there was a difference.  The string "PDM 1.0" didn't pass validation, so I changed it to CC0. I'm not sure if the publisher chose with purpose either, let's review the [source](https://nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/4fa03182-df71-4c39-87da-e7d5c0b82d88):

-  **Usage limitations**:  No usage limitations
- **Licenses**: http://creativecommons.org/publicdomain/mark/1.0/deed.nl 
http://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse/noConditionsApply
http://inspire.ec.europa.eu/metadata-codelist/LimitationsOnPublicAccess/noLimitations

The data is also published as [WFS](https://service.pdok.nl/rvo/referentiepercelen/wfs/v1_0?request=GetCapabilities&service=WFS), and here license http://creativecommons.org/publicdomain/zero/1.0/deed.nl is used (CC0).

As the data is officially published under all of these licenses, we can choose a suitable one. CC0 sound simplest to me. Should I document this, or do you have a different take on this?

Reply:
Thanks for the investigations. Maybe just put the comment you just wrote as a comment in the code.
So the official code according to [SPDX](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-list/) is `CC0-1.0` (the validation is very simple and allows CC0 although not in the SPDX list).